src/rpc/blockchain.cpp

```diff
@@ -1496,16 +1496,76 @@ UniValue MempoolInfoToJSON(const CTxMemPool& pool)
     ret.pushKV("mempoolminfee", ValueFromAmount(std::max(pool.GetMinFee(maxmempool), ::minRelayTxFee).GetFeePerK()));
     ret.pushKV("minrelaytxfee", ValueFromAmount(::minRelayTxFee.GetFeePerK()));
 
+    if (with_fee_histogram) {
```

Comment:
Maybe move this directly into `getmempoolinfo`? Or another helper?

Reply:
I thought about another call, but extending `mempoolinfo` with an option for "more data" seems to be most allied with other calls where one can get more extended infos on option.

Reply:
I mean just have the code outside this function. The RPC would then call both `MempoolInfoToJSON` and also `JSONMempoolInfoAddHistogram` (or whatever this code gets called)